joomla/joomla-cms
# [5.2] [Guided tours] Tour header visual changes + footer padding fix

### Summary of Changes

The guided tours header does not have a clear indication that a tour can be cancelled. There is no label and the progress indicator is next to it, making it hard to distinguish.
The changes include:
- A _Cancel_ label close to the cancel cross to provide context and enhance usability,
- moving the progress indicator away from the cancel button to ensure the cancel is clearly visible.

### Testing Instructions

It is just a visual change. Make sure you can still cancel a tour, no matter if you are selecting the cross or the label. 

### Actual result BEFORE applying this Pull Request

![image](https://github.com/user-attachments/assets/2d582c4e-1652-4d45-b30a-b452b7a555bd)

### Expected result AFTER applying this Pull Request

![image](https://github.com/user-attachments/assets/06eea5e5-7780-4be4-9866-df99ab87b8c9)

### Link to documentations
Please select:
